tools/scenario-player/scenario_player/main.py

```diff
@@ -103,9 +104,14 @@ def main(scenario_file, keystore_file, password, rpc_url, auth, mailgun_api_key)
         if isinstance(handler, logging.StreamHandler):
             handler.stream = log_buffer
             break
+
     with open(keystore_file, 'r') as keystore:
         account = Account(json.load(keystore), password, keystore_file)
         log.info("Using account", account=to_checksum_address(account.address))
+
+    # Collect tasks
+    collect_tasks(tasks)
```

Comment:
Nitpick: Maybe this would be better placed in `ScenarioRunner.__init__`. But since we're most likely not going to re-use this anywhere else anyway this is fine as well.

Reply:
Problem with this is that the `ScenarioRunner` module is imported by the `base.py` module containing `Task`. This would lead to a circular dependency.